src/auth/jwt-interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface JwtPayload {
+  userId: string;
+  email: string;
+}
```

Comment:
PullProofAI

The `JwtPayload` interface in `jwt-interface.ts` defines the structure of the JWT payload with `userId` and `email` properties. It's clear and concise.

This change introduces a well-defined interface for the JWT payload, enhancing type safety and readability in authentication-related code. No issues found.



<!-- This is an auto-generated comment by PullProofAI -->